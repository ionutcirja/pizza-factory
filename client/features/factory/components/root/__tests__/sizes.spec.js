import React from 'react';
import { shallow } from 'enzyme';
import Sizes from '../sizes';

describe('Sizes component', () => {
  const propsToRender = {
    options: [
      'option 1',
      'option 2',
      'option 3',
      'option 4',
    ],
  };
  
  describe('render', () => {
    it('should render a select form field component', () => {
      const wrapper = shallow(<Sizes {...propsToRender} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
