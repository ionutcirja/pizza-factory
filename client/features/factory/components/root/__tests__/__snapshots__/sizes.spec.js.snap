// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sizes component render should render a select form field component 1`] = `
<FormField
  label="Please select your desired size"
  name="size"
  options={
    Array [
      "option 1",
      "option 2",
      "option 3",
      "option 4",
    ]
  }
  placeholder="Select pizza size"
  type="select"
/>
`;
